README.md

```diff
@@ -83,8 +83,8 @@ In addition to the model equations, the initial conditions and parameters are al
 ```Julia
   using DifferentialEquations, Plots
 
-  sol = solve(prob)
-  plot(sol, vars=(1,3))
+  sol = solve(prob, RK4())    # fourth-order Runge-Kutta method
```

Comment:
No tutorials should use RK4

Reply:
I just wanted something generic. The default algorithm choice needs DifferentialEquations.jl and is unavailable in OrdinaryDiffEq. Should we change it to DifferentialEquations.jl or use a different solver? It is for the Lorenz equation that can probably be solved with anything.

Reply:
Using DifferentialEquations.jl is generally a good idea for tutorials as we want people using the default algorithm for most things because they tend to not make the right choices.